pkgs/development/interpreters/python/hooks/propagate-wrapper-args.sh

```diff
@@ -0,0 +1,33 @@
+# Hook that propagates wrapper arguments up to the
+# python interpreter buildEnv environment.
+echo "Sourcing propagate-wrapper-args.sh"
+
+storeWrapperArgsHook() {
+    # Store the arguments for use by other derivations
+    # and the python interpreter derivation.
+    echo "Executing propagateWrapperArgsHook"
+
+    mkdir -p "$out/nix-support"
+    if [ ''${#qtWrapperArgs[@]} -ne 0 ]; then
```

Comment:
is the `''$` escaping necessary within a .sh file?

Reply:
Nope, I must have forgot to remove it once I moved the code to an external hook. Thanks.